web/GETTING_STARTED.md

```diff
@@ -19,53 +19,65 @@ This page provides info how to use Web OTE tool. Before following these instruct
+To achieve better results you may use your data to fine-tune pre-trained models for a specific class of objects. 
 1. Choose the model and press `Fine-tune`.
-2. Give a name to the model, select the needed build and the number of epochs. If you want to save val images with drawn predictions on it select `Save annotated val images`. 
-3. It is better to select advanced settings and select the batch size. The higher batch size value helps to train a more general model and achieve better accuracy. But too high value may provoke the memory problem: `ERROR: Unexpected bus error encountered in worker. This might be caused by insufficient shared memory (shm)`. 
-You may conduct several attempts to chose the highest valid batch size or simply make it equal to 1.
+2. Give a name to the model, select the needed build and the number of epochs. If you want to save val images with drawn predictions on it, select `Save annotated val images`. 
+3. It is better to select advanced settings and tune the batch size. The higher batch size value helps to train a more general model and achieve better accuracy. However, too high value may provoke the memory problem: `ERROR: Unexpected bus error encountered in worker. This might be caused by insufficient shared memory (shm)`. 
+You may conduct several attempts to choose the highest valid batch size or simply make it equal to 1.
 4. Be ready that training will start with a little delay.
 5. During the training you may monitor the progress with logs or Tensorboard graphs.
 
 # Custom detection
-If you did not find a suitable trained model for your task you may create the Custom detection task. In this case, you will use models pre-trained on the great number of classes that are able to extract features from images. So, fine-tuning during several epochs will help you to train the model to detect specific or rare classes.  
+If you did not find a suitable trained model for your task, you may create the Custom detection task. In this case, you will use models pre-trained on the great number of classes that are able to extract features from images. So, fine-tuning during several epochs will help you to train the model to detect specific or rare classes.  
 1. Select `Create custom detection task` on the main page.
 2. Fill all related information including classes you want to detect. 
 3. Put your pictures in `<training_extensions>/web/data/assets` folder, annotate it and create a build.
 4. Select a model from a list (more complex models are better for detecting small objects) and fine-tune it. Be ready that it may take more epochs than for standard object detection tasks to get good results.
 
 # Known issues and problems 
 1. Both train and val parts should contain at least 60 images for proper work.
-2. Horizontal text detection model can not be evaluated and fine-tuned on CPU.
+2. Horizontal text detection model can be fine-tuned on GPU only.
```

Comment:
There is a controversial point where to locate the problem with Text detection fine-tuning on CPU. 
From one point of view, it is not WEB UI problem, it is an architectural feature, so it cannot be fixed by Dmitriy or Vitaliy. 
From another point of view, it will be useful for a user to be aware of it before setup, so it should be located in `Known issues` section.